xonsh/tools.py

```diff
@@ -1686,15 +1686,29 @@ def globpath(s, ignore_case=False, return_empty=False):
 
 def _iglobpath(s, ignore_case=False):
```

Comment:
That should work, a normal optional argument probably would be easier.
Still, I am unsure whether it would be good style to introduce such an argument just for testing.
Do you think that this would be alright?